config/urls.py

```diff
@@ -15,14 +15,15 @@
 admin.site.site_url = settings.SITE_URL 
 
 
-urlpatterns = patterns(
-    '',
+urlpatterns = [
     # Uncomment the next line to enable the admin:
     url(r'^jet/', include('jet.urls', 'jet')),
     url(r'^jet/dashboard/', include('jet.dashboard.urls', 'jet-dashboard')),  # Django JET dashboard URLS
-    url(r'^admin/', include(admin.site.urls)),
+    url(r'^admin/', admin.site.urls),
```

Comment:
Why was `include()` removed from the admin?

Reply:
In Django 1.9+, you don't need to use include

Reply:
Gotcha! Thanks